### ISSUE TITLE
0.18.3 trulens_eval fails when importing OpenAI

### ISSUE THREAD

**Issue by benke-rl** [truera/trulens#677]
I've upgraded llama_index to the latest 0.9.15 due to an [incompatibility](https://github.com/truera/trulens/issues/675) in trulens_eval 0.18.2 which is fixed in 0.18.3. However, 0.18.3 now fails when importing OpenAI:

```
Traceback (most recent call last):
    from trulens_eval import OpenAI
  File "/venv/lib/python3.9/site-packages/trulens_eval/__init__.py", line 83, in <module>
    from trulens_eval.feedback import Bedrock
  File "/venv/lib/python3.9/site-packages/trulens_eval/feedback/__init__.py", line 16, in <module>
    from trulens_eval.feedback.feedback import Feedback
  File "/venv/lib/python3.9/site-packages/trulens_eval/feedback/feedback.py", line 17, in <module>
    from trulens_eval.feedback.provider.endpoint.base import Endpoint
  File "/venv/lib/python3.9/site-packages/trulens_eval/feedback/provider/__init__.py", line 1, in <module>
    from trulens_eval.feedback.provider.base import Provider
  File "/venv/lib/python3.9/site-packages/trulens_eval/feedback/provider/base.py", line 8, in <module>
    from trulens_eval.feedback.provider.endpoint.base import Endpoint
  File "/venv/lib/python3.9/site-packages/trulens_eval/feedback/provider/endpoint/__init__.py", line 6, in <module>
    from trulens_eval.feedback.provider.endpoint.openai import OpenAIEndpoint, OpenAIClient
  File "/venv/lib/python3.9/site-packages/trulens_eval/feedback/provider/endpoint/openai.py", line 47, in <module>
    class OpenAIClient(SerialModel):
  File "/venv/lib/python3.9/site-packages/trulens_eval/feedback/provider/endpoint/openai.py", line 68, in OpenAIClient
    client: Optional[oai.OpenAI | oai.AzureOpenAI] = None,
TypeError: unsupported operand type(s) for |: 'type' and 'type'
```

**Comment by joshreini1** [comment]
Hey @benke-rl ! We merged a fix for this earlier this week and will release it with 0.19 tomorrow.

If you don’t want to wait, feel free to install from the repo. Thanks!

**Comment by joshreini1** [comment]
Following up @benke-rl  - this change is released! Please upgrade to trulens_eval==0.19.1. Thanks!
